docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>I feel that the concept of <strong>core case</strong> came out of nowhere. Maybe we could introduce it more explicitly. Something like:</p><p> </p><blockquote>To understand what is going on, we need to introduce the concept of parameter core dimensions. The core dimensions of a distribution's parameter are the minimum number of dimensions the parameters need to have, in order to make the distribution be well defined. In the Multinomial distribution, <code>n</code> must at least be an scalar integer, but <code>p</code> must be at least a vector that represents the probability of having an outcome on each category. So, for the Multinomial distribution, <code>n</code> has 0 core dimensions, and <code>p</code> has 1 core dimension.  </blockquote><p>You can then continue from "So if we have a vector..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
Done